README.md

```diff
@@ -68,7 +84,15 @@ metrics.register_callback(function()
 end)
 ```
 
-Using histograms:
+### Histogram
+
+Histogram samples observations and counts them into configurable buckets.
+Histograms are useful for tracking request latencies, processing time.
+Histogram is exposed as multiple numerical values: 
+- the total count of observed events
+- the total sum of observed values
+- per bucket counters of observed events
```

Comment:
`counters of observed events per bucket`

Reply:
done